### PR TITLE
Updated Dependencies and Unit Tests

### PR DESCRIPTION
**Pull request by NikCharlebois**
#### Pull Request (PR) description
* Updated Unit Tests to remove the logic to install all dependencies.
* Updated the Power Apps dependencies to version 2.0.159

#### This Pull Request (PR) fixes the following issues
* N/A

### CODE REVIEW

File: .github/workflows/Unit Tests.yml [hunk 1]
```diff
@@ -19,30 +19,7 @@ jobs:
...
-              {
-                Install-Module $module.ModuleName -RequiredVersion $module.RequiredVersion -Force -Scope AllUsers
-              }
-              catch
-              {}
-            }
-          }
+          Install-Module ReverseDSC -Force -Scope AllUsers
           Install-Module PSDesiredStateConfiguration -Force -Scope AllUsers
           Install-Module Pester -Force -SkipPublisherCheck -RequiredVersion 5.4.0 -Scope AllUsers
```
Comment (ykuijs) [inline .github/workflows/Unit Tests.yml:24]: v5.4.1 has just been released

**Review by ykuijs — COMMENTED** [review]
Minor comment

**Comment by NikCharlebois** [comment]
I am working on updating the Stubs Generator to fix the errors. Should take another 15 minutes or so.

**Review by ykuijs — APPROVED** [review]
LGTM

**Pull request merged**